grails-app/services/de/iteratec/osm/measurement/environment/wptserver/WptInstructionService.groovy

```diff
@@ -89,7 +89,7 @@ class WptInstructionService {
         Map paramsNotNull = parameters.findAll { k, v -> v != null }
         def result = null
         log.debug("Try to launch job at wptserver=${wptServer} with params=${parameters}")
-        performanceLoggingService.logExecutionTime(DEBUG, "Launching job ${job.label}: Calling initial runtest on wptserver.", 1) {
+        performanceLoggingService.logExecutionTime(DEBUG, "Launching job ${job.label}: Calling initial runtest on wptserver.", PerformanceLoggingService.IndentationDepth.ONE) {
```

Comment:
Couldn't you remove the `PerformanceLoggingService.` part?

Reply:
This occurs in multiple files.

Reply:
I'll change it.